classloader/src/main/java/io/smallrye/common/classloader/DefineClassPermission.java

```diff
@@ -0,0 +1,11 @@
+package io.smallrye.common.classloader;
+
+import java.security.BasicPermission;
+
+public class DefineClassPermission extends BasicPermission {
```

Comment:
Don't extend `BasicPermission` because the permission class defines no actions.

Reply:
Done. Let me know if the new implementation is better :)